data/friends.yml

```diff
@@ -36,6 +36,11 @@
   avatar: https://wlanxww.com/images/avatar.jpeg
   url: https://wlanxww.com
   description: wlanxww's blog
+  
+   -nickname: i'm ajie
```

Comment:
There is a few problems with your commit:
1. Incorrect formatting and indentation: You should add your FixIt site data in nickname lexicographical order.
2. Your website is unavailable, Please check again!

Reply:
ok